pullpreview.gemspec

```diff
@@ -0,0 +1,18 @@
+Gem::Specification.new do |spec|
```

Comment:
Do we want to make this a gem?

Reply:
yeah, I think the maintainer was actually liking the idea of doing so